lexicons/com/atproto/identity/signPlcOp.json

```diff
@@ -0,0 +1,41 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.identity.signPlcOp",
+  "defs": {
+    "main": {
+      "type": "procedure",
+      "description": "Signs a PLC operation to update some value(s) in the requesting DID's document.",
+      "input": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "properties": {
+            "rotationKeys": {
+              "type": "array",
+              "items": { "type": "string" }
+            },
+            "alsoKnownAs": {
+              "type": "array",
+              "items": { "type": "string" }
+            },
+            "verificationMethods": { "type": "unknown" },
+            "services": { "type": "unknown" }
+          }
+        }
+      },
+      "output": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "required": ["plcOp"],
+          "properties": {
+            "plcOp": {
```

Comment:
Maybe we should sync this property name up with the input of `identity.sendPlcOp` (i.e. both "op" or both "plcOp").

Reply:
yup agreed 👍